lib/ifc/analysis-utils.ts

```
@@ -1,89 +1,155 @@
+            expressId: e.expressId,
+            type: e.type,
+            GlobalId: e.properties?.GlobalId
+          })),
+          metric
+        },
+        [arrayBuffer] // Transfer ArrayBuffer to avoid copying
+      )
+    } catch (err) {
+      reject(err)
+    }
+  })
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)